crates/uv/src/commands/python/install.rs

```diff
@@ -136,6 +137,11 @@ pub(crate) async fn install(
 ) -> Result<ExitStatus> {
     let start = std::time::Instant::now();
 
+    if default && !preview.is_enabled() {
+        writeln!(printer.stderr(), "The `--default` flag is only available in preview mode; add the `--preview` flag to use `--default.")?;
```

Comment:
Nit: I think we only use a period when a message spans multiple sentences

Reply:
I considered this but the semi-colon... haha I'll remove it.